src/ui/components/cards/accordioncardcomponent.js

```diff
@@ -143,13 +143,13 @@ export default class AccordionCardComponent extends Component {
   }
 
   onMount () {
-    const toggleEl = DOM.query(this._container, '.js-yxt-AccordionCard-toggle');
-    const accordionBodyEl = DOM.query(this._container, '.js-yxt-AccordionCard-body');
     const accordionEl = DOM.query(this._container, '.js-yxt-AccordionCard');
-
-    accordionBodyEl.style.height = `${this.isExpanded ? accordionBodyEl.scrollHeight : 0}px`;
-
-    DOM.on(toggleEl, 'click', () => this.handleClick(toggleEl, accordionBodyEl, accordionEl));
+    if (!(this._config.details == null || this._config.details === '')) {
```

Comment:
I think we can just do `if (!this._config.details)` here!
@alexisgrow might remember better, but I think the really weird looking `this.details = this.details === null ? null : (this.details || result.details || rawResult.details || '');` was written with regards to some oddly specific QA item

Reply:
Changed it to if (this._config.details)!